pulp_2_tests/tests/puppet/api_v2/test_duplicate_uploads.py

```diff
@@ -14,24 +14,35 @@
 .. _Pulp #1406: https://pulp.plan.io/issues/1406
 .. _Pulp Smash #81: https://github.com/PulpQE/pulp-smash/issues/81
 """
-from pulp_smash import api, utils
-from pulp_smash.pulp2.constants import REPOSITORY_PATH
-from pulp_smash.pulp2.utils import BaseAPITestCase, DuplicateUploadsMixin
+import unittest
+
+from pulp_smash import api, utils, config
+from pulp_smash.pulp2.constants import REPOSITORY_PATH, ORPHANS_PATH
+from pulp_smash.pulp2.utils import DuplicateUploadsMixin
 
 from pulp_2_tests.constants import PUPPET_MODULE_URL_1
 from pulp_2_tests.tests.puppet.api_v2.utils import gen_repo
 from pulp_2_tests.tests.puppet.utils import set_up_module as setUpModule  # pylint:disable=unused-import
 
 
-class DuplicateUploadsTestCase(BaseAPITestCase, DuplicateUploadsMixin):
+class DuplicateUploadsTestCase(unittest.TestCase, DuplicateUploadsMixin):
     """Test how well Pulp can deal with duplicate content unit uploads."""
 
     @classmethod
     def setUpClass(cls):
         """Create a Puppet repository."""
-        super().setUpClass()
+        cls.cfg = config.get_config()
+        cls.resources = set()
         unit = utils.http_get(PUPPET_MODULE_URL_1)
         import_params = {'unit_type_id': 'puppet_module'}
         repo = api.Client(cls.cfg).post(REPOSITORY_PATH, gen_repo()).json()
```

Comment:
```suggestion
        cls.client(cls.cfg, api.json_handler)
        repo = cls.client.post(REPOSITORY_PATH, gen_repo())
```

This will allow reuse the same client on `tearDownClass`.